specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/loadBalancer.json

```diff
@@ -1226,7 +1226,7 @@
           },
           "description": "Gets load balancing rules that use this backend address pool."
         },
-        "outboundRule": {
+        "outboundRules": {
```

Comment:
`outboundRule` definition was for a single resource, so you would need to add additional data to indicate that `outboundRules` is an array. But I don't think you should remove `outboundRule` since this could be a breaking change (and I can see that property on backend model)